Makefile

```diff
@@ -86,7 +86,7 @@ kolibri.apk: p4a_android_distro src/kolibri needs-version
 	@echo "--- :android: Build APK"
 	p4a apk --release --sign --arch=$(P4A_ARCH) --version=$(APK_VERSION) --numeric-version=$(BUILD_NUMBER)
 	mkdir -p dist
-	mv kolibri__$(P4A_ARCH)-$(APK_VERSION)-.apk dist/kolibri__$(P4A_ARCH)-$(APK_VERSION).apk
+	mv kolibri-$(APK_VERSION)-.apk dist/kolibri__$(ARM_VER)-$(APK_VERSION).apk
```

Comment:
When I run this this original file now includes `-release-` in it. I think that comes from gradle. So, this line should be:
```
mv kolibri-release-$(APK_VERSION)-.apk dist/kolibri__$(ARM_VER)-$(APK_VERSION).apk
```

Reply:
```suggestion
	mv kolibri-release-$(APK_VERSION)-.apk dist/kolibri__$(ARM_VER)-$(APK_VERSION).apk
```